components/Buttons/src/MDCButton.h

```diff
@@ -268,6 +290,10 @@
  */
 - (nullable UIColor *)shadowColorForState:(UIControlState)state;
 
+/** When @c YES, the imageTintColor forState: APIs are enabled. Defaults to @c NO. */
+@property(nonatomic, assign, getter=isImageTintStatefulAPIEnabled) BOOL imageTintStatefulAPIEnabled;
```

Comment:
Can this be an internal private flag that gets turned on when we call setImageForTintColor:forState:?

If not we should move this next to the other API so that they are grouped